transcom/mymove
# B-21548 Update service items BE

## [B-21548](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1022468)

## Summary

This PR sets pre approved service items for international shipments.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Login as a TOO and add an HHG international shipment or shipments to a move*
2. Check off the shipment(s) and click "Approve selected"
3. Confirm in the service items preview modal that the shipment has 4 service items:
    International Shipping & Linehaul - ISLH
    Fuel Surcharge - POEFSC, or PODFSC (if the shipment has a CONUS origin or destination)
    International HHG Pack - IHPK
    International HHG Unpack - IHUPK
4. Click "Approve and send" on the modal
5. Confirm on the move task orders details page that the service items are added under the shipment
6. Confirm that you can accept and reject these pre approved service items

*Alternatively, you can make a move and add an international HHG shipment. Approve it as a service counselor, and then follow steps 2-6

### Backend


## Screenshots
_**Preview of pre approved service items**_
<img width="1094" alt="Screenshot 2024-11-29 at 7 23 17 AM" src="https://github.com/user-attachments/assets/b2d8a1f2-4684-4bc9-a783-aabbc410694d">

_**TOO approving or rejecting service items**_
<img width="1018" alt="Screenshot 2024-11-29 at 7 23 43 AM" src="https://github.com/user-attachments/assets/8117f050-81c9-4214-9fd3-4cfe2bc4c7c4">
